standard/operator.lua

```diff
@@ -85,4 +85,24 @@ function Operator.method(funcName, ...)
 	end
 end
 
+---@generic T
+---@param a T
+---@return T
+function Operator.identity(a)
```

Comment:
There's already FnUtil.identity, what's the plan for it?

Reply:
removed it again, didn't remember the FnUtil one